config/initializers/dartsass.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.config.dartsass.builds = {
```

Comment:
This is from https://github.com/rails/dartsass-rails?tab=readme-ov-file#configuring-builds